.github/workflows/rust.yml

```diff
@@ -30,6 +30,28 @@ jobs:
       - name: Run tests
         run: cargo test
 
+  push_package_demo:
```

Comment:
Happy to move this away from `rust.yml`, since it's not entirely "rust-related", although it is used by `kubit` for its tests.

I think either works :smile: